functions/src/models/user.ts

```diff
@@ -21,7 +21,7 @@ export enum UserType {
 
 export interface User {
   type: UserType
-  dateOfBirth?: string
+  dateOfBirth?: string | null
```

Comment:
Models that have optional fields instead of `| null`:

`Invitation, User, UserMessagesSettings`